extension/src/experiments/model/modify/collect.test.ts

```diff
@@ -8,10 +8,10 @@ describe('collectFlatExperimentParams', () => {
     const params = collectFlatExperimentParams(rowsFixture[0].params)
     expect(params).toStrictEqual([
       { path: appendColumnToPath('params.yaml', 'code_names'), value: [0, 1] },
-      { path: appendColumnToPath('params.yaml', 'epochs'), value: 2 },
+      { path: appendColumnToPath('params.yaml', 'epochs'), value: 5 },
       {
         path: appendColumnToPath('params.yaml', 'learning_rate'),
-        value: 2.2e-7
+        value: 2.1e-7
```

Comment:
[F] Changed the data in the workspace to match the experiment now running in the workspace in the test fixture